projects/helmet/helmet_project.py

```diff
@@ -42,9 +51,19 @@ def condition_func(self, total_results):
         person_model1 = self.mapping[person_model0][1]  # Mapping person from model1 to model0
         helmet_model0 = 1
 
-        return (any(box.cls == person_model0 for box in total_results[0].boxes)
-                and any(box.cls == helmet_model0 for box in total_results[0].boxes)
-                and any(box.cls == person_model1 for box in total_results[1].boxes))
+        has_person_model0 = any(box.cls == person_model0 for box in total_results[0].boxes)
+        has_helmet_model0 = any(box.cls == helmet_model0 for box in total_results[0].boxes)
+        has_person_model1 = any(box.cls == person_model1 for box in total_results[1].boxes)
+        has_minimum_width_height_model0 = all(box.xywh[0, 2] > self.min_width
+                                              and box.xywh[0, 3] > self.min_height for box in total_results[0].boxes
+                                              if box.cls == person_model0)
+        has_minimum_width_height_model1 = all(box.xywh[0, 2] > self.min_width
+                                              and box.xywh[0, 3] > self.min_height for box in total_results[1].boxes
```

Comment:
I believe we should also add the `max_width` and `max_value` so we make sure we are harvesting objects of the same size? What do you think?